srcpkgs/python-pywatchman/template

```diff
@@ -0,0 +1,34 @@
+# Template file for 'python-pywatchman'
+pkgname=python-pywatchman
+version=1.4.1
+revision=1
+wrksrc="pywatchman-${version}"
+build_style=python-module
+only_for_archs="x86_64 x86_64-musl"
+pycompile_module="pywatchman"
+hostmakedepends="python-devel python3-devel python-setuptools python3-setuptools"
+makedepends="curl"
```

Comment:
to fetch the LICENSE file

Reply:
not actually required, you can just add the path to the raw license in the distfiles= in this manner

distfiles="url-to-first-thing
 https://raw.githubusercontent.com/facebook/watchman/master/python/LICENSE>LICENSE.txt"

.txt files are merely extracted into the XBPS_BUILDDIR, you can then `vlicense ../LICENSE.txt` to install it.